portal/src/assets/style/_mobile.scss

```diff
@@ -2,544 +2,605 @@
+		flex-direction: column;
+	}
+
+	.mobile-home-tx-table tr:not(:nth-child(-n + 5)) {
+		display: none;
+	}
+
+	/* home page transaction count chart */
+	.mobile-latest-transactions-label {
+		font-size: 1.1rem;
+		font-weight: bold;
+	}
+
+	/* latest transactions */
+
+	.mobile-wrapper-tx-table > .tx-table {
+		margin-left: 1rem;
+	}
+
+	.mobile-wrapper-tx-table > .tx-table th {
+		padding-left: 0 !important;
+		font-weight: bold;
+		font-size: 1rem;
+		border-top: none;
+	}
+
+	.mobile-wrapper-tx-table > .tx-table th:first {
+		padding-left: 0;
+		font-weight: bold;
+		font-size: 1rem;
+		border-top: none;
+	}
+
+	.mobile-wrapper-tx-table > .tx-table td {
+		padding-left: 0;
+	}
+
+	/********** home page transaction count chart **********/
+	.mobile-latest-transactions-label {
+		font-size: 1.1rem;
+		font-weight: bold;
+	}
+
+	/********** home page latest transactions table **********/
+	.mobile-home-tx-table > .tx-table {
+		margin-left: 1rem;
+	}
+
+	.mobile-home-tx-table > .tx-table th {
+		padding-left: 0 !important;
+		font-weight: bold;
+		font-size: 1rem;
+		border-top: none;
+	}
+
+	.mobile-home-tx-table > .tx-table th:first {
+		padding-left: 0;
+		font-weight: bold;
+		font-size: 1rem;
+		border-top: none;
+	}
+
+	.mobile-home-tx-table > .tx-table td {
+		padding-left: 0;
+	}
+
+	.mobile-no-box-shadow {
+		box-shadow: none;
+	}
+
+	.mobile-home-view-all-btn {
+		flex: 0.3 !important;
+	}
+
+	.mobile-home-tx-table tr:not(:nth-child(-n + 5)) {
+		display: none;
+	}
 }
 
 .mobile-show-important {
-  @media only screen and (max-width: $mobile-width) {
-    display: block !important;
-  }
-  display: none !important;
+	@media only screen and (max-width: $mobile-width) {
+		display: block !important;
+	}
+	display: none !important;
 }
 
 .scroll-bar-wrapper {
-  @media only screen and (max-width: $mobile-width) {
-    display: block;
-    border: none 0px;
-
-    overflow-x: scroll;
-    overflow-y: hidden;
-    height: 20px;
-    padding: 0 1rem 0 1rem;
-    margin: 5px;
-
-    &::-webkit-scrollbar {
-      -webkit-appearance: none;
-    }
-    &::-webkit-scrollbar:horizontal {
-      height: 17px;
-    }
-    &::-webkit-scrollbar-thumb {
-      background-color: rgba(0, 0, 0, .5);
-      border-radius: 5px;
-      border: 2px solid #ffffff;
-    }
-    &::-webkit-scrollbar-track {
-      background-color: #ECECEC;
-      border-radius: 3px;
-    }
-  }
-  display: none;
+	@media only screen and (max-width: $mobile-width) {
+		display: block;
+		border: none 0px;
+
+		overflow-x: scroll;
+		overflow-y: hidden;
+		height: 20px;
+		padding: 0 1rem 0 1rem;
+		margin: 5px;
+
+		&::-webkit-scrollbar {
+			-webkit-appearance: none;
+		}
+		&::-webkit-scrollbar:horizontal {
+			height: 17px;
+		}
+		&::-webkit-scrollbar-thumb {
+			background-color: rgba(0, 0, 0, 0.5);
+			border-radius: 5px;
+			border: 2px solid #ffffff;
+		}
+		&::-webkit-scrollbar-track {
+			background-color: #ececec;
+			border-radius: 3px;
+		}
+	}
+	display: none;
 }
```

Comment:
This block of code should not of changed. Please use the same indentation and formatting  of project.  You have not added any code to the  .scroll-bar-wrapper block